### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by paibamboo**
Update all packages

### Test fails on Redux Saga 1.1.0 and 1.1.1 so 1.0.5 is installed instead
After this [pr](https://github.com/redux-saga/redux-saga/pull/1892), there are errors when running `npm run test`
> StarsSaga › fetchStarsCount › registerListeners › listens for loadStarsCount INVOKED and calls fetchStarsCount

and

> SettingsSaga › registerListeners › listens for setLanguage INVOKED and calls fetchTranslations

According to this Stackoverflow [answer](https://stackoverflow.com/questions/53162001/typeerror-during-jests-spyon-cannot-set-property-getrequest-of-object-which/53307822#53307822), `jest.spyOn` will not work if the module that it spies on re-exports another module. But he says it works when using `ts-jest` or `tsc`. So not sure what's wrong now.

### React Helmet causes React componentWillMount warning
Watching this [issue](https://github.com/nfl/react-helmet/issues/465)

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -30,11 +30,13 @@
     "no-string-based-set-timeout": true,
     "react-no-dangerous-html": [
       true,
-      {
-        "file": "./src/app/containers/Html.tsx",
-        "method": "render",
-        "comment": "string comes from trusted source - our initial state"
-      }
+      [
```
Comment (cyberhck) [inline tslint.json]: wow, the fix was easy 😄 I didn't look internally

**Comment by cyberhck** [comment]
nice, btw, I also installed this: https://github.com/apps/semantic-pull-requests (commit message validation with an app)

File: tslint.json (hunk 1 above)
Comment (paibamboo) [reply tslint.json]: Hey I took this [fix](https://github.com/microsoft/tslint-microsoft-contrib/issues/880#issue-463671661) from you. Come on Nishchal.

Comment (cyberhck) [reply tslint.json]: wow 😄 I forgot about it.

**Pull request merged**